### DISCUSSION TITLE
Aseprite

### DISCUSSION THREAD

**Discussion by Ankh64** [catppuccin/catppuccin#1863] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Comment by DakshG07** [comment]
Would be cool :/

**Comment by lrxyab** [comment]
are libresprite ones compatible?
i can try making something for libresprite ig

**Comment by Ankh64** [comment]
yes i believe so

**Comment by Ankh64** [comment]
i'm currently editing the using the dase theme sheet as a base with some modifications, let me know if everyone's okay with that theme.

**Comment by Muvkina** [comment]
Are you still working on this? If so any repository to check out? I'm interested in helping out

**Comment by Ankh64** [comment]
Yes! I am, although its stagnated a bit recently.. I need help making a build script to generate all the themes from a base theme, The repo I'm working in is currently private since its nowhere near ready to publish, also sorry for the late response.

**Comment by Ankh64** [comment]
The mocha-mauve version!
![image](https://user-images.githubusercontent.com/85212821/189264798-7d905af1-0018-45a5-b822-b943b3845950.png)

**Comment by Muvkina** [comment]
Awesome! Unfortunately some stuff has come out so I can't help anymore at the moment. But awesome work.

**Comment by lrxyab** [comment]
hey you can try out using [my script](https://github.com/skinatro/theme-tool)
sorry i have been busy  with exams, so i havent added a proper readme  yet

**Comment by Ankh64** [comment]
omg thanks i've got 3 out of the 4 themes done after using your script, its really convenient.

**Comment by Muvkina** [comment]
No rush but I now have some more time and was wondering if the repo is public yet?

**Comment by Muvkina** [comment]
Since it's been a while without a public repo i may start working on a port. One of the few applications i use missing a port lol

**Reply by Ankh64** [reply]
hey, join us on ctp discord, we have a forum post on this, i have just lost interest in this project.

**Comment by sofycat** [comment]
Released here
https://github.com/sofycat/aseprite/releases/tag/v1.0.0

**Reply by sgoudham** [reply]
Hiya :wave: @sofycat  

This looks great! Would you be willing to raise an issue to get this reviewed by staff and transferred into the organisation? 
You can use the issue template given [here](https://github.com/catppuccin/catppuccin/issues/new?assignees=&labels=&projects=&template=port-review.yml&title=Name+of+the+application%2Ftool%2Fwebsite%2Fetc.)!

**Comment by sgoudham** [comment]
Closing as discussion has moved to #2126
